float64 hip_joint
float64 thigh_joint
float64 calf_joint